docs/setup-cluster/k8s/resource-caps.rst

```diff
@@ -0,0 +1,135 @@
+.. _k8s-resource-caps:
+
+#####################################
+ Manage Workspace Namespace Bindings
+#####################################
+
+.. note::
+
+   Auto creating namespaces and managing resource quotas in Determined require the Determined
+   Enterprise Edition.
+
+Determined makes it easier to manage resource quotas by handling them directly, so you don't need to
+manage them separately in Kubernetes. You can manage workspace namespace bindings and resource
+quotas using either the WebUI or CLI.
+
+*******
+ WebUI
+*******
+
+#. As an admin user, either create a new workspace or edit an existing workspace.
+
+#. In the "Namespace Bindings" section, type a name for the namespace or leave it blank. The
+   behavior depends on the master configuration. For example, if the ``default_namespace`` is
+   defined, leaving the namespace blank binds the workspace to the "default" namespace (even if it
+   is not set) configured for that Kubernetes cluster. If a different namespace name is provided,
+   the workspace is bound to that namespace.
+
+#. Toggle the "Auto Create Namespace" option on or off. When enabled, the system automatically
+   creates a namespace in the cluster, allowing you to edit the resource quota directly in
+   Determined.
+
+#. Enter a resource quota (e.g., 5). This quota is editable only when "Auto Create Namespace" is on.
+   If the quota is set in Kubernetes, it will appear here but won't be editable.
```

Comment:
Technically, even if someone edits a Determined-created quota in Kubernetes, they can still come back and set it again in Determined. Maybe we should reword this to say "If the quota for a namespace is created in Kubernetes, it will appear here but won't be editable".

We should also probably note that we display the lowest quota on the namespace, so if that's a Determined-created quota, the user can edit it in the WebUI and if the lowest quota was created in Kubernetes, then it won't be editable.